R/module_star_systems.R

```diff
@@ -101,6 +105,19 @@ star_systems_server = function(id, data) {
       show_hz = shiny::reactive(plot_options()$show_hz)
     )
 
+    system_info = shiny::reactive(system_data() %>%
+      select(hostname, sy_dist, sy_snum, sy_pnum, st_teff, st_rad, st_mass, st_lum) %>%
+      dplyr::slice(1) %>%
+      mutate(st_teff = classify_star_spectral_type(st_teff)) %>%
+      mutate(
+        across(c(sy_dist, st_rad, st_mass, st_lum), ~ round(.x, 3))
+      ) %>%
+      rename("Stellar spectral class" = st_teff) %>%
+      rename(any_of(exoplanets_col_labels[["pscomppars"]]))
+    )
+
```

Comment:
style: Multi-line blocks of code should be wrapped in `{}`:
```suggestion
    system_info = shiny::reactive({
      system_data() %>%
        select(hostname, sy_dist, sy_snum, sy_pnum, st_teff, st_rad, st_mass, st_lum) %>%
        dplyr::slice(1) %>%
        mutate(st_teff = classify_star_spectral_type(st_teff)) %>%
        mutate(across(c(sy_dist, st_rad, st_mass, st_lum), ~ round(.x, 3))) %>%
        rename("Stellar spectral class" = st_teff) %>%
        rename(any_of(exoplanets_col_labels[["pscomppars"]]))
    })

```